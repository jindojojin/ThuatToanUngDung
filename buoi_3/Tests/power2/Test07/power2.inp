12
1 16 1555 23456 6789654 1234567 4444444 1073741824 1000 17757 234577 14588
