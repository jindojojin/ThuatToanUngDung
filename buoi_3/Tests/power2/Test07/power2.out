1 16 1073741824
